src/mqt/bench/utils.py

```diff
@@ -342,3 +343,73 @@ def postprocess_single_oqc_file(filename: str):
 
 def create_zip_file():
     return subprocess.call(f"zip -rj {get_zip_file_path()} {get_qasm_output_path()}", shell=True)
+
+
+def calc_qubit_index(qargs: list[Any], qregs: list[QuantumRegister], index: int) -> Any:
```

Comment:
Why are you annotating the return type with `Any` here? Shouldn't that just be `int`?

Reply:
Fixed that.